FirebaseAuth/Tests/Sample/AuthSample.xcodeproj/project.pbxproj

```diff
@@ -3,11 +3,15 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 50;
```

Comment:
run `pod deintegrate`

Reply:
Nope, unintentionally checked the `FirebaseAuth/Tests/Sample/` directory in. Reverted.